pkg/gatherers/clusterconfig/ceph_cluster.go

```diff
@@ -16,27 +16,32 @@ import (
 // GatherCephCluster Collects statuses of the`cephclusters.ceph.rook.io` resources
 // from Openshift Data Foundation Stack.
 //
-// ### API Reference:
-// https://github.com/rook/rook/blob/master/pkg/apis/ceph.rook.io/v1/types.go
+// ### API Reference
+// - https://github.com/rook/rook/blob/master/pkg/apis/ceph.rook.io/v1/types.go
 //
-// ### Sample data:
-// docs/insights-archive-sample/config/storage/openshift-storage/cephclusters/ocs-storagecluster-cephcluster.json
+// ### Sample data
+// - docs/insights-archive-sample/config/storage/openshift-storage/cephclusters/ocs-storagecluster-cephcluster.json
 //
-// ### Location in archive:
-// | Version | Path													 |
-// | ------- | ----------------------------------------------------- |
-// | <= 4.12 | config/storage/{namespace}/{name}.json 				 |
-// | 4.13+   | config/storage/{namespace}/cephclusters/{name}.json 	 |
+// ### Location in archive
+// | Version   | Path													 |
+// | --------- | ----------------------------------------------------- |
+// | <= 4.12.0 | config/storage/{namespace}/{name}.json 				 |
+// | 4.13.0+   | config/storage/{namespace}/cephclusters/{name}.json 	 |
```

Comment:
So what would be the path for e.g 4.12.4 ?

Reply:
It is also helpful if rule developers could know the backport version of the path change.

Reply:
@shzhou12 not sure if I understood it. What do you mean by that?

Are you saying that, if we backport the changes from `4.13.0` that changes to `4.12.40`, and introduce the change of path by doing it we should also add it as a note? If yes, in this case, we just need to change the version in the table that points to a new path (line 29) to be `>= 4.12.40`

Reply:
I mean this. The pattern will be more complicated if the file path changes backported to multiple versions. If we do not backport file path changes, please ignore my comment.

Reply:
Well, I can't assure you that we will not backport path changes... So, I guess that now I can see the problem here.
  
Let's imagine that we have a feature that is currently changing the path, and at first glance we considered:

| Version   | Path  |
| --------- | ------- |
| < 4.13.0 | config/old/path/file.json 	|
| >= 4.13.0 | config/new/path/file.json	 |

Later then, we decide to backport this change to `4.12.4` and `4.11.10`, and we update the table:

| Version   | Path  |
| --------- | ------- |
| < 4.11.10 | config/old/path/file.json 	|
| >= 4.11.10 | config/new/path/file.json	 |

In this case, we do have a problem. Mostly 'cause we don't have the change for some `>= 4.12.0 < 4.12.4`. So, basically, it will not work.

Even my idea to add a note, will not help... So, we will probably have to specify, in this case, every single backported version that is affected by the change in the table.

| Version   | Path  |
| --------- | ------- |
| < 4.11.10 | config/old/path/file.json 	|
| 4.11.10+ | config/new/path/file.json	 |
| >= 4.12.4 | config/new/path/file.json	 |

This probably overlaps with the discussion that we are having [here](https://github.com/openshift/insights-operator/pull/704#discussion_r1067072076)

Reply:
OK, seems that mixing `<= < > >=` with `+` isn't something nice. So, in the best-case scenario, we will have to specify in the table any change of path that could happen in backports, to leave it as clear as possible. I'll review all and update the docs again.